delivery_tnt_oca/models/tnt_request.py

```diff
@@ -162,7 +162,11 @@ def _quant_package_data_from_picking(self):
             "LENGTH": data_total["length"],
             "HEIGHT": data_total["height"],
             "WIDTH": data_total["width"],
-            "WEIGHT": data_total["weight"],
+            "WEIGHT": (
+                data_total["weight"] / self.record.number_of_packages
```

Comment:
Shouldn't it be always the division?

Reply:
Yes, you are right, because it is not sent in PACKAGE the package listing (if used).